### PR TITLE
Fix reply icon on answered messages

### PR DESCRIPTION
**Pull request by GretaD**
For Christoph's eyes only

replydark
replylight

### CODE REVIEW

File: src/components/Envelope.vue [hunk 1]
```diff
@@ -455,7 +456,6 @@ list-item-style.draft .app-content-list-item-line-two {
 }
 
 .icon-reply {
-	background-image: url('../../img/reply.svg');
 	-ms-filter: 'progid:DXImageTransform.Microsoft.Alpha(Opacity=50)';
 	opacity: 0.5;
```
Comment (GretaD) [inline src/components/Envelope.vue:460]: should i change the opacity .25 same as attachment?

Comment (ChristophWurst) [reply src/components/Envelope.vue:460]: Whatever makes the look the same :wink:

File: src/components/Envelope.vue [hunk 2]
```diff
@@ -455,9 +456,8 @@ list-item-style.draft .app-content-list-item-line-two {
 }
 
 .icon-reply {
-	background-image: url('../../img/reply.svg');
```
Comment (ChristophWurst) [inline src/components/Envelope.vue:458]: where's the icon loaded from if this is dropped? :thinking:

Comment (GretaD) [reply src/components/Envelope.vue:458]: web console says its from mail.css

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**